src/pinyin_converter.coffee

```diff
@@ -9,69 +9,71 @@ email: dchanin@quizlet.com
 
-    convert: (string) ->
-        # Find words with a number behind them, and replace with callback fn.
-        matches = string.match @pinyinRegex
-        return string unless matches
+	convert: (string) ->
+		# Find words with a number behind them, and replace with callback fn.
+		matches = string.match @pinyinRegex
+		return string unless matches
 
-        for match in matches
-            replacement = @getReplacement(match)
-            string = string.replace(match, replacement)
-        string
+		for match in matches
+			replacement = @getReplacement(match)
+			string = string.replace(match, replacement)
+		string
 
-    ########################## Private #####################################
+	########################## Private #####################################
 
-    getReplacement: (match) ->
-        accentMap = @getAccentMap()
-        tone = match.slice(-1)
-        word = match.slice(0, -1).replace('v', 'ü').replace('V', 'Ü')
-        for base, vowel of accentMap
-            if word.indexOf(base) >= 0
-                vowelChar = vowel.match(/.\*/)[0]
-                vowelNum = @vowels[vowelChar]
-                accentedVowelChar = @pinyin[tone.toString()][vowelNum]
-                replacedWord = word.replace(base, vowel).replace(vowelChar, accentedVowelChar)
-                return replacedWord
-        match
+	getReplacement: (match) ->
+		accentMap = @getAccentMap()
+		tone = match.slice(-1)
+		word = match.slice(0, -1).replace('v', 'ü').replace('V', 'Ü')
+		for base, vowel of accentMap
+			if word.indexOf(base) >= 0
+				vowelChar = vowel.match(/.\*/)[0]
+				vowelNum = @vowels[vowelChar]
+				accentedVowelChar = @pinyin[tone.toString()][vowelNum]
+				replacedWord = word.replace(base, vowel).replace(vowelChar, accentedVowelChar)
+				return replacedWord
+		match
 
-    getAccentMap: ->
-        unless @accentMap
-            stars =
-                'a*i a*o e*i ia* ia*o ie* io* iu* ' +
-                'A*I A*O E*I IA* IA*O IE* IO* IU* ' +
-                'o*u ua* ua*i ue* ui* uo* üe* ' +
-                'O*U UA* UA*I UE* UI* UO* ÜE* ' +
-                'A* E* I* O* U* Ü* ' +
-                'a* e* i* o* u* ü*'
-            nostars = stars.replace(/\*/g, '')
-            starsArray = stars.split(' ')
+	getAccentMap: ->
+		unless @accentMap
+			stars =
+				'a*i a*o e*i ia* ia*o ie* io* iu* ' +
+				'A*I A*O E*I IA* IA*O IE* IO* IU* ' +
+				'o*u ua* ua*i ue* ui* uo* üe* ' +
+				'O*U UA* UA*I UE* UI* UO* ÜE* ' +
+				'A* E* I* O* U* Ü* ' +
+				'a* e* i* o* u* ü*'
+			nostars = stars.replace(/\*/g, '')
+			starsArray = stars.split(' ')
 
-            @accentMap = {}
-            for base, i in nostars.split(' ')
-                @accentMap[base] = starsArray[i]
-        @accentMap
\ No newline at end of file
+			@accentMap = {}
+			for base, i in nostars.split(' ')
+				@accentMap[base] = starsArray[i]
+		@accentMap
+
+module.exports = PinyinConverter if module?.exports?
```

Comment:
Couldn't hurt to just make this [UMD compatible](http://ricostacruz.com/cheatsheets/umdjs.html#no-dependencies), seems to me.